bosh.yml

```diff
@@ -192,3 +214,16 @@ variables:
     common_name: default.hm.bosh-internal
     extended_key_usage:
     - client_auth
+
+- name: blobstore_ca
```

Comment:
Are we concerned about the name of the variables changing? This will generate new credentials and not necessarily be a zero downtime deploy. Are we concerned about upgrades that use an experimental ops file?